### PR TITLE
[medDropSites] set fixed sizes

### PR DESCRIPTION
**Pull request by mathildemerle**
Wonderful, less code, and a better ergonomy :')

I applied the same as in https://github.com/LoicCadour/medInria-public/pull/70 to fix the medDropSites sizes.

:m:

### CODE REVIEW

**Comment by LoicCadour** [comment]
Fine for me.

**Comment by mathildemerle** [comment]
Hehe ok Loic i just removed the 2nd thumbnail in medMaskApplication :p

**Comment by LoicCadour** [comment]
Fine!

File: src-plugins/medMaskApplication/medMaskApplicationToolBox.cpp [hunk 1]
```diff
@@ -44,59 +43,42 @@ class medMaskApplicationToolBoxPrivate
...
-    d->imageDropSite->setCanAutomaticallyChangeAppereance(true);
+    connect (d->maskDropSite,   SIGNAL(objectDropped(const medDataIndex&)), this, SLOT(importMask(const medDataIndex&)));
+    bundlingLayout->addWidget(d->maskDropSite);
 
     QPushButton *clearMaskButton = new QPushButton("Clear mask", widget);
     clearMaskButton->setToolTip(tr("Clear previously loaded mask."));
-    QPushButton *clearImageButton = new QPushButton("Clear Image", widget);
-    clearImageButton->setToolTip(tr("Clear previously loaded image."));
+    clearMaskButton->setFixedWidth(d->maskDropSite->sizeHint().width());
+    connect (clearMaskButton,   SIGNAL(clicked()), this, SLOT(clearMask()));
```
Comment (fcollot) [inline src-plugins/medMaskApplication/medMaskApplicationToolBox.cpp]: excess spaces here

File: src-plugins/medMaskApplication/medMaskApplicationToolBox.cpp [hunk 2]
```diff
@@ -141,116 +123,39 @@ medAbstractData* medMaskApplicationToolBox::processOutput()
...
-    {
-        d->process= new medMaskApplication;
-    }
-    d->data = data;
-    
-    if (!d->process)
-    {
-        return;
+        medJobManager::instance()->registerJobItem(runProcess);
+        QThreadPool::globalInstance()->start(dynamic_cast<QRunnable*>(runProcess));
```
Comment (fcollot) [inline src-plugins/medMaskApplication/medMaskApplicationToolBox.cpp]: This cast is not needed since medRunnable derived from QRunnable

**Comment by fcollot** [comment]
Aside from my comments, I'm ok with this

**Comment by mathildemerle** [comment]
Updated, thx for the review :)

**Pull request merged**